packages/vendor/package.json

```diff
@@ -39,26 +39,26 @@
     "postpublish": "npm run deploy:docs"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
-    "@babel/preset-env": "^7.7.0",
+    "@babel/core": "^7.15.8",
+    "@babel/preset-env": "^7.15.8",
     "@theforeman/vendor-core": "^8.16.0",
-    "babel-loader": "^8.0.6",
-    "compression-webpack-plugin": "^2.0.0",
+    "babel-loader": "^8.2.3",
+    "compression-webpack-plugin": "^6.1.1",
     "copy-webpack-plugin": "^4.6.0",
-    "css-loader": "^2.1.1",
-    "file-loader": "^5.1.0",
-    "lodash": "^4.17.14",
-    "mini-css-extract-plugin": "^0.5.0",
-    "node-sass": "^4.12.0",
+    "css-loader": "^3.6.0",
+    "file-loader": "^6.2.0",
+    "lodash": "^4.17.21",
+    "mini-css-extract-plugin": "^1.6.2",
+    "node-sass": "^6.0.1",
```

Comment:
node 10 is not supported here https://github.com/sass/node-sass/releases/tag/v6.0.0